### DISCUSSION TITLE
LF APAC 开源布道者分享系列之七：如何娴熟使用异步沟通之法 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#22] [Announcements]
# LF APAC 开源布道者分享系列之七：如何娴熟使用异步沟通之法 - 开源之道

https://opensourceway.community/posts/lfapac-evangelist-sharing/async-in-the-open-source-community/

**Comment by ZoomQuiet** [comment]
#是也乎, (￣▽￣)

参考: [邮件列表的规范和礼节：创造良性发展的交流空间](http://s5.zoomquiet.top/050730-usMaillist/index.html)
这是 2005年, 作为 douban 第1号员工, 洪教授在 BPUG 会课上的一次分享,

那之后, 俺就有了一个偏见: `容器/形式决定内容`

异步/同步沟通, 其实都是沟通, 本身并无优劣之分,
但是, 在不同场景/容器中, 才引发了效能问题,

比如, 在没有互联网时代, 用邮件不能同步沟通嘛? 当然可以, 看你用什么客户端;
在移动互联时代, 用微信不能异步沟通嘛? 当然也可以, 比如双方分布在相差12个小时的不同时区,
基本上, 每次消息, 都要大半天后, 才能收到回复...

问题是, `容器/形式决定内容`,
在微信界面中, 很难进行复杂, 有结构的陈述...

所以, 什么是异步沟通的需求?

不过是生活节奏罢了, 如果领导要求今天必须完成, 那么就算是对面现在是凌晨4点, 也一定想办法吼起来直播联调的吧....

异步不是创新需求, 只不过是足够创作空间的现实反射而已...

**Reply by lijiangsheng1** [reply]
生活的节奏，对生命的理解，决定了和他人协作的方式。再往深挖就是个人主义与集体主义之争。
